contents/tutorials/limit-session-recordings.md

```diff
@@ -0,0 +1,148 @@
+---
+title: How to only record the sessions you want
+date: 2023-03-22
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-17.png
+topics: ["session recordings", "configuration"]
+---
+
+Session recordings help you get a deep understanding of how users are using your product. They are our top recommendation for [early-stage startups](/blog/early-stage-analytics), but as you scale the number of recordings can go beyond what you need.
+
+Instead of turning session recordings off entirely, you can use PostHog’s configuration options to only record the sessions you want. This tutorial shows you how.
```

Comment:
```suggestion
Instead of turning session recordings off entirely, you can use PostHog’s configuration options to only record the sessions you want. This tutorial shows you three ways to do this. 
```